webapp/package.json

```diff
@@ -49,6 +49,7 @@
     "@types/jasminewd2": "~2.0.3",
     "@types/node": "~8.9.4",
     "codelyzer": "~4.5.0",
+    "cross-fetch": "^3.0.4",
```

Comment:
It is only used in dynacred, not in webserver, so it shouldn't be in devDeps

Reply:
but if I do `make dynacred_src`, then I need it. But only in dev, not in production, as the latter doesn't (shouldn't) depend on `make dynacred_src`...

Reply:
Ha right, good for me then